src/KubernetesClient/Kubernetes.WebSocket.cs

```diff
@@ -239,11 +239,6 @@ protected async Task<WebSocket> StreamConnectAsync(Uri uri, string webSocketSubP
             }
 
             // Set Credentials
-            if (this.ClientCert != null)
```

Comment:
Why are you deleting this?

Reply:
This is dead code. `this.ClientCert` is never assigned and is always null before the change. I did a git blame and it seems it's a leftover from a previous refactoring. The logic is covered by the code blow.